frontend/src/service-panel/ServicePanel.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import Button from './Button';
+import VpTitle from './VpTitle';
+import VpInfo from './VpInfo';
+import './ServicePanel.less';
+
+interface ServicePanelProps {
+  isVisible: boolean;
+  onButtonClick(e: React.MouseEvent): void;
+}
+
+const ServicePanel: React.StatelessComponent<ServicePanelProps> = ({ isVisible, onButtonClick }) => (
+  <div>
+    <Button onClick={onButtonClick} />
+    <VpTitle />
+    <VpInfo isVisible={isVisible} />
```

Comment:
`VpInfo` is not quite exact title for the component. Something like `Panel` will be better IMO.